src/css/catalog/CatalogPage.css

```
@@ -0,0 +1,38 @@
+/* Mobile first approach. Responsive breakpoints are listed smallest to highest */
+/* Small devices, Phones up to 575px */
+body {
+  margin-top: 55px;
+  background-color: var(--darkgrey);
+  font-family: var(--fontfamily);
+  font-size: 20px;
+}
+
+.catalog-page{
+  display: grid;
+  grid-template-columns: 1fr 27rem;
+}
+
+.filterHeader, .coursesHeader{
```

Consistency. All the other css classes are kabob case, so these should be too
```suggestion
.filter-header, .courses-header{
```